### ISSUE TITLE
Support for Image posts.

### ISSUE THREAD

**Issue by AkinBilgic** [krisives/cronnit.us#6]
Would love to have the ability to schedule Image posts as well.  But imgur will do for now.  Thanks!

**Comment by krisives** [comment]
Thanks for the kind donation. I'll see if I can reverse how Reddit does the image upload or at the very least automate the process of uploading to Imgur. The only reason it hasn't been done yet is because Reddit doesn't actually document how their image upload features work or make it "officially" part of their API.

I'll keep this issue updated with my findings.

**Comment by Convoy20** [comment]
Would simply allowing link posts be the best way to approach this?

**Comment by krisives** [comment]
@Convoy20 Link posts are allowed. Simply use a URL as the text and it will be submitted as a link post.

**Comment by donkarziavelli** [comment]
Hey, any new infos on the possibility of image posts? :)

**Comment by matthewstrasiotto** [comment]
I'd like to close this @MLanghof  , I don't intend to implement this, and I doubt anyone else does. 

Personally I find the workflow of imgur / gfycat upload , and link posting good-enough, and not disimilar to common workflows even without cronnit.

**Comment by MLanghof** [comment]
Yes, submitting links works, but that was already the case when this issue was opened. I don't think anything has changed substantially since then (other than Kris having little time at the moment).
